src/spikeinterface/qualitymetrics/misc_metrics.py

```diff
@@ -261,22 +261,28 @@ def compute_isi_violations(sorting_analyzer, isi_threshold_ms=1.5, min_isi_ms=0,
     Returns
     -------
     isi_violations_ratio : dict
-        The isi violation ratio described in [Hill]_.
+        The isi violation ratio.
     isi_violation_count : dict
         Number of violations.
 
     Notes
     -----
-    You can interpret an ISI violations ratio value of 0.5 as meaning that contaminating spikes are
-    occurring at roughly half the rate of "true" spikes for that unit.
-    In cases of highly contaminated units, the ISI violations ratio can sometimes be greater than 1.
+    The returned ISI violations ratio measures the approximate fraction of spikes in each
+    unit which are contaminted. This interpretation is good when the ratio is small, and
+    becomes worse as it grows. In cases of highly contaminated units, the ISI  violations
```

Comment:
This is 100% clear. Worse being what exactly... "As it grows past 0.xx it begins to diverge from xx" something like that based on llobet's comments in the issue.

Reply:
Mmm. I was deliberately vague because the success of the approximation depends on lots of factors: when the contamination is large is it because there are LOTS spikes near the same time a few times, or many violations of where one spike violates only one other spike? The Llobet comment is on one specific example (uniformly generated random spikes) so you shouldn't take precise numbers from it. 

All the models agree when $n_vT/N^2t_$ is small (much less than one), so we could say that the interpretation only works in that situation.

Reply:
That's fair. Okay so maybe something more like:

This intrepretation is good when the ratio is small, but as the ratio approaches 1 it is a less reliable indicator (or something like that? ). I know the ratio can go above one so maybe we don't even want that. But "becomes worse" is he vague part for me. ie becomes worse at what?

Reply:
I've realised that's also not true: you can cook up examples (where each spike is contaminated by exactly one other spike) where the ratio is a reliable indicator. So we can only really say: sometimes this is a good indicator, sometimes it isn't (like most of the quality metrics!). I suggest we refer to the Llobet paper, which discusses some of the different models? I've had a go.